lib/CallBackery/qooxdoo/callbackery/source/class/callbackery/ui/form/Auto.js

```diff
@@ -127,7 +127,7 @@ qx.Class.define("callbackery.ui.form.Auto", {
             if (s.set) {
                 if (s.widget == 'date') {
                     if (s.set.value) {
-                        var dateValue = s.set.value;
+                        var dateValue = s.set.value * 1000; // incoming epoch seconds
                         delete s.set.value;
```

Comment:
On Thu, 1 Oct 2015, Tobias Oetiker wrote:

>> @@ -127,7 +127,7 @@ qx.Class.define("callbackery.ui.form.Auto", {
>>              if (s.set) {
>>                  if (s.widget == 'date') {
>>                      if (s.set.value) {
>> -                        var dateValue = s.set.value;
>> +                        var dateValue = s.set.value * 1000; // incoming epoch seconds
>>                          delete s.set.value;
>
> ok ...  before it was possible to pass a string which would have
> initialized the date object too.  now input must be an integer ...  fine
> with me

Good point. I guess I could check if it is a number and only then multiply
it.

Fritz

-- 
Oetiker+Partner AG              tel: +41 62 775 9903 (direct)
Fritz Zaucker                        +41 62 775 9900 (switch board)
Aarweg 15                            +41 79 675 0630 (mobile)
CH-4600 Olten                   fax: +41 62 775 9905
Schweiz                         web: www.oetiker.ch